winui/Rating/Tooltip.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: ToolTip with WinUI Rating control | Syncfusion
+description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
```

Comment:
description: Learn all about Enabletooltip and tooltip format support in the rating items in WinUI rating control here.

Reply:
Changed the content description as mentioned above.